packages/tools/kadena-cli/src/account/tests/accountFund.test.ts

```diff
@@ -103,6 +103,15 @@ describe('account fund', () => {
     );
   });
 
+  it('should throw max amount error when user tries to enter more than max amount split for multi chains', async () => {
+    const res = await runCommand(
+      'account fund --account=account-add-test-manual --amount=20 --network=testnet --chain-ids=0,1,2 --quiet',
+    );
+    expect(res.stderr).toContain(
+      'Error: -m, --amount "With 3 chains to fund, the max amount per chain is 6 coin(s)."',
```

Comment:
Are they referred to as "coin(s)" anywhere else? the term sounds new to me but I might be wrong

Reply:
it's not directly used as `coin(s)` -> but as `${fungible}(s)`